.github/workflows/pr_basic_compilation_check.yml

```diff
@@ -32,8 +32,13 @@ jobs:
       - name: Compile "enclave-tls"
         run:
           cd enclave-tls;
-          source /root/.bashrc && make && make install && make clean &&
-            make SGX=1 && make clean && make OCCLUM=1
+          wget https://download.01.org/intel-sgx/sgx-linux/2.13.3/as.ld.objdump.gold.r3.tar.gz;
+          tar -xf as.ld.objdump.gold.r3.tar.gz;
```

Comment:
It is better to install as.ld.objdump.gold.r3.tar.gz to develop images. The Dockerfile of develop images is in 
`https://github.com/alibaba/inclavare-containers/tree/master/tools/docker`